pyvrp/cpp/educate/LocalSearch.cpp

```diff
@@ -272,13 +272,7 @@ void LocalSearch::loadIndividual(Individual const &individual)
 {
     for (size_t client = 0; client <= data.numClients(); client++)
     {
-        clients[client].tw = {static_cast<int>(client),  // TODO cast
-                              static_cast<int>(client),  // TODO cast
-                              data.client(client).serviceDuration,
-                              0,
-                              data.client(client).twEarly,
-                              data.client(client).twLate};
-
+        clients[client].tw = {client, data.client(client)};
```

Comment:
Can we also rename `Node::tw` to `Node::tws` (and also for `Node::twBefore` and `Node::twAfter`?

Reply:
> Can we also rename `Node::tw` to `Node::tws` (and also for `Node::twBefore` and `Node::twAfter`?

Not opposed, but I suggest we leave this as-is. I'm planning to refactor `Node` and `Route` quite a bit in a few weeks, and this will probably change as part of that.